environment/src/main/java/com/sequenceiq/environment/environment/service/sdx/SdxService.java

```diff
@@ -87,6 +87,22 @@ public void stopByCrn(String crn) {
         }
     }
 
+    public boolean isStoppable(String crn) {
+        try {
+            String initiatorUserCrn = ThreadBasedUserCrnProvider.getUserCrn();
+            return ThreadBasedUserCrnProvider.doAsInternalActor(
+                    regionAwareInternalCrnGeneratorFactory.iam().getInternalCrnForServiceAsString(),
+                    () -> sdxEndpoint.isStoppableInternal(crn, initiatorUserCrn)
+            );
+        } catch (WebApplicationException e) {
+            String errorMessage = webApplicationExceptionMessageExtractor.getErrorMessage(e);
+            LOGGER.error(String.format(
```

Comment:
Warn level would be better here as from the application/service point of view this exception could occur in case of a simple network issue.

Reply:
Makes sense, I'll do that then, thank you.